support/functions.ds

```diff
@@ -1,4 +1,4 @@
-# Usage: generate_generic <feature_tests or example> <backend name> <output folder name> <optional docs folder> <optional additional args>
+# Usage: generate_generic <feature_tests or example> <backend name> <output folder name> <optional docs folder> <optional additional args> <optional subdirectory for copy>
```

Comment:
nit: document how the optional subdirectory on a separate line

Reply:
I added a much more detailed explanation of the optional subdirectory